frontend/screens/BrandRegistrationForm.js

```diff
@@ -1,187 +1,206 @@
-import * as React from "react";
+import React, { useState } from "react";
 import { Image } from "expo-image";
-import { StyleSheet, View, Text, ScrollView, TouchableOpacity } from "react-native";
+import {
+  StyleSheet,
+  View,
+  Text,
+  ScrollView,
+  TouchableOpacity,
+  TextInput,
+  Alert,
+} from "react-native";
 import { Color, Padding, FontSize, Border, FontFamily } from "../GlobalStyles";
-import { useNavigation } from "@react-navigation/native";
 
+const BrandRegistrationForm = ({ navigation }) => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [brandType, setBrandType] = useState("");
+  const [username, setUsername] = useState("");
 
-const BrandRegistrationForm = () => {
-  const navigation = useNavigation();
+  const handleSubmit = async () => {
+    // Construct the payload
+    const payload = {
+      email,
+      password,
+      category: [brandType],
+      name: username,
+    };
+    if (!email || !password || !brandType || !username) {
```

Comment:
can we please add email and password validation? Also, how are we making sure that username is unique?

Reply:
added